pkg/odo/cli/build_images/build_images.go

```diff
@@ -58,7 +58,7 @@ func (o *BuildImagesOptions) Validate() (err error) {
 }
 
 // Run contains the logic for the odo command
-func (o *BuildImagesOptions) Run() (err error) {
+func (o *BuildImagesOptions) Run(cmdline cmdline.Cmdline) (err error) {
```

Comment:
If we pass the `cmdline` in `Run`, I'm afraid we will be using it for bad reasons. I would prefer we pass the context only, which is ok and good practice to pass to all methods.

Reply:
Updated